activemodel/CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+*   `serializable_hash` has a new option called rename.
+
+    it can be used when you want to rename attributes or methods.
+
+    ```ruby
+    class Person
+      include ActiveModel::Serialization
+
+      attr_accessor :name, :age
+
+      def attributes
+        {'name' => nil, 'age' => nil}
+      end
+    end
+
+    person.serializable_hash(rename: {name: :first_name}) # => {"first_name"=>"bob", "age"=>22}
```

Comment:
As this is the changelog, I think we can be more terse and let the reader guess more context -- if it sounds interesting, they can check the documentation.

```suggestion
    ```ruby
    person.serializable_hash(only: %w(name age), rename: { name: :first_name })
      # => { "first_name" => "bob", "age" => 22 }
```